executor/load_data.go

```diff
@@ -241,7 +241,7 @@ func (e *LoadDataInfo) CommitWork(ctx context.Context) error {
 		case <-e.QuitCh:
 			err = errors.New("commit forced to quit")
 			logutil.Logger(ctx).Error("commit forced to quit, possible preparation failed")
-			break
+			return err
```

Comment:
It seems that the  `tasks` in line 237 is redundant, please remove it.

Reply:
It used at line 252.

Reply:
But it does nothing, this is an useless usage.

Reply:
I think this val is used for counting the task count and logged on line 255.
Did you mean we do not need count this?

Reply:
Ok, my mistake.